README.adoc

```diff
@@ -44,15 +46,93 @@ If you still don't know what to do with e2e tests in some use-cases, go to <<Wha
+
+1. Download CodeReady Containers from https://developers.redhat.com/products/codeready-containers/overview[developers.redhat.com].
+You will need to login using your Red Hat SSO account, after which you may click on the `Install OpenShift on your laptop` button which will take you to the download page for CRC.  From here, select your OS before clicking `Download CodeReady Containers`.  You will also need to download your pull secret, and keep that in a safe place.
++
+image::download.png[align="center"]
+
+2. Extract the downloaded file into a directory of your choice.
++
+image::extract_crc.png[align="center"]
++
+You may also wish to add this directory to your path so that you can run the `crc` command from anywhere.
+
+3. Enable Cluster Monitoring, which is required for tests to pass by running the following command:
++
+`crc config set enable-cluster-monitoring true`
+
+4. Since cluster monitoring requires at least 14GB of virtual machine memory, you also need to increase the default by running the following command - the value here is sufficient but you may make it higher if desired:
++
+`crc config set memory 14500`
+
+5. Set the `QUAY_NAMESPACE` environment variable to your quay username:
++
+`export QUAY_NAMESPACE=<quay-username>`
+
+6. Ensure that the following Quay repositories are set to public:
++
+* host-operator
+* member-operator
+* member-operator-webhook
+* registration-service
+
+You can find these on the https://quay.io/repository/[Repository] page at quay.io:
+
+image::quay_repo.png[align=center]
+
+Click on the repository name, then click on the settings icon:
+
+image::quay_repo_detail.png[align=center]
+
+Scroll down to the *Repository Visibility* section, and ensure the repository is set to public:
+
+image::quay_repo_visibility.png[align=center]
+
+7. Start the CRC container by executing `crc start`. This will take a few minutes to complete, and the output should look something like this:
+
+image::crc_start_output.png[align=center]
+
+Take note of the `oc login...` command shown in the output.
+
+8. Execute the following command to set up the necessary environment variables:
+
+`eval $(crc oc-env)`
+
+9. Log into the cluster by executing the `oc login` command as shown in the resulting output from running `crc start` in the previous step.  The output should look something like this:
++
+image::crc_oc_login.png[align=center]
+
+10. Check out the CodeReady Toolchain `host-operator` and `member-operator` repositories and ensure that they are located at the same directory level as the `toolchain-e2e` repository.
++
+* https://github.com/codeready-toolchain/host-operator
+* https://github.com/codeready-toolchain/member-operator
++
+11. Change into the `toolchain-e2e` directory, and execute the tests.  It is recommended that you direct console output to a text file, as the test output is of significant length and is likely to be longer than your console buffer:
```

Comment:
Personally, I've configured my terminal to have an unlimited buffer for the scrolling :-)

Reply:
I output to a file too. It's easier to search and navigate in a file. But it's all personal preferences :)

Reply:
Yes, sure - I didn't mean that it should be removed :-). I just mentioned it in case Shane wouldn't be aware of this option :wink: 
The README part can stay as it is :+1: